queries/walletBalances/useTokensBalancesQuery.ts

```diff
@@ -0,0 +1,82 @@
+import { Provider, Contract } from 'ethcall';
```

Comment:
`ethcall` is a library based on MakerDAO's `multicall` to make multiple requests in one go (https://www.npmjs.com/package/ethcall)

There are various libraries out there that do the same, this one works nicely with ethers.